lib/PhasarLLVM/TypeHierarchy/LLVMVFTable.cpp

```diff
@@ -48,4 +50,33 @@ nlohmann::json LLVMVFTable::getAsJson() const {
   return J;
 }
 
+std::vector<const llvm::Function *>
+LLVMVFTable::getVFVectorFromIRVTable(const llvm::ConstantStruct *VT) {
+  std::vector<const llvm::Function *> VFS;
+  for (const auto &Op : VT->operands()) {
+    if (auto *CA = llvm::dyn_cast<llvm::ConstantArray>(Op)) {
+      for (auto It = CA->operands().begin() + 2; It != CA->operands().end();
+           ++It) {
+        auto &COp = *It;
+        if (auto *CE = llvm::dyn_cast<llvm::ConstantExpr>(COp)) {
+          if (auto *BC = llvm::dyn_cast<llvm::BitCastOperator>(CE)) {
+            // if the entry is a GlobalAlias, get its Aliasee
+            auto *ENTRY = BC->getOperand(0);
```

Comment:
Why is `ENTRY` spellend in upper case letters? Is this an exception to the naming schema?

Reply:
Yeah, this needs to be altered - this code fragment is code that existed previously and was refactored into this static function, see rest of diff. Nevertheless, the name is bad.